changelogs/fragments/8289-python-runner.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - PythonRunner module utils - specialisation of ``CmdRunner`` to execute python scripts (https://github.com/ansible-collections/community.general/pull/8289).
```

If you don't mean the `python` CLI program (then you should use double backticks), Python should be capitalized:
```suggestion
  - PythonRunner module utils - specialisation of ``CmdRunner`` to execute Python scripts (https://github.com/ansible-collections/community.general/pull/8289).
```